src/main/java/edition/academy/seventh/controller/BookController.java

```diff
@@ -0,0 +1,34 @@
+package edition.academy.seventh.controller;
+
+import edition.academy.seventh.database.model.Book;
+import edition.academy.seventh.serivce.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/** @author Kamil Rojek */
+@RestController
+@CrossOrigin(origins = "*", maxAge = 3600)
+class BookController {
+  private BookService bookService;
+
+  @Autowired
+  public BookController(BookService bookService) {
+    this.bookService = bookService;
+  }
+
+  /**
+   * Gets the list of the books.
```

Comment:
powinno być "Retrieves list of books." Pytanie jakie książki? Wszystkie? Pasowałoby to wyszczególnić